src/main/CommandHandler.cpp

```diff
@@ -768,6 +770,107 @@ CommandHandler::scpInfo(std::string const& params, std::string& retStr)
     retStr = root.toStyledString();
 }
 
+Json::Value
+makeQuorumSetNode(stellar::SCPQuorumSet const& value) {
+    Json::Value node;
+    node["threshold"] = value.threshold;
+
+    Json::Value validators;
+    for (auto const& key : value.validators) {
+        validators.append(KeyUtils::toShortString(key));
```

Comment:
I think toStrKey would be better than toShortString here - it does not provide enough information to identify node

Reply:
Done.